bots/migrations/0127_backfill_run_conversation.py

```
@@ -0,0 +1,143 @@
+
+def _disable_auto_timestamps(RunConversation):
+    for name in ("created_at", "updated_at"):
+        field = RunConversation._meta.get_field(name)
+        field.auto_now = False
+        field.auto_now_add = False
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("bots", "0126_runconversation_savedrun_conversation_and_more"),
+    ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)